packages/plugin-kafka/README.md

```diff
@@ -86,3 +84,21 @@ for await (const message of consumer) {
   // process message
 }
 ```
+
+## Helpers
+
+### KafkaConsumerStream
+
+#### consumer.allMessagesRead(timeout: number): boolean
+This helper method checks whether maximum offsets reached for all assigned partitions in subscribed topics. The `timeout` parameter is required and passed to the `node-rdkafka` internals.
+This helper added because `node-rdkafka` omits `EOF` messages by default and this normal behavior.
+
+##### Example
+
+```js
+for await (const message of consumer) {
+  if (await consumer.allMessagesRead(300)) {
```

Comment:
should wrap consumer stream's drain message and improve `read` method of the stream, where it would validate EOF

Reply:
If we enable this helper for all ConsumerStreams, there will be great performance penalty because we perform query for each assigned partition watermarks. Thats why I try to make it optional function. Also Consumer stream as Readable is not emitting`drain` event.